_includes/nav-page-menu.html

```diff
@@ -818,6 +818,7 @@
               <li class="hide-mobile"><a href="/libraries/">Libs</a></li>
               <li><a href="/rsk/node/architecture/json-rpc/">JSON-RPC</a></li>
               <li><a href="https://web3js.readthedocs.io/" target="_blank">Web3 Spec</a></li>
+              <li><a href="https://docs.ethers.io/v5/" target="_blank">ethers.js</a></li>
```

Comment:
```suggestion
              <li><a href="https://docs.ethers.io/" target="_blank">ethers.js</a></li>
```

to be forward compatible with newer versions when they are released